magefiles/dependency.go

```diff
@@ -17,7 +17,7 @@ const (
 	kindVersion        = "0.20.0"
 	yqVersion          = "4.35.1"
 	goimportsVersion   = "0.12.0"
-	olmVersion         = "0.20.0"
+	olmVersion         = "0.27.0"
```

Comment:
Lets revert this back to 0.20.0. With this back to 0.20.0 both our (@venkateshsredhat ) tests are passing locally.